admin_guide/installation.rst

```diff
@@ -203,6 +207,46 @@ Note that you'll still need to follow the CryptPad configuration steps, especial
       - ./onlyoffice-conf:/cryptpad/onlyoffice-conf
       - ./config/config.js:/cryptpad/config/config.js
 
+.. _admin_install_onlyoffice_manually:
+
+Install OnlyOffice manually
+~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+It is easier to use the :ref:`script <admin_install_onlyoffice>` to install OnlyOffice. However, it is also possible to install OnlyOffice manually.
+
+For the first installation you need to clone `onlyoffice-builds` into your `cryptpad` folder:
+
+.. code:: bash
+
+   git clone --bare https://github.com/cryptpad/onlyoffice-builds.git
+
+After that, you can check out the different OnlyOffice versions CryptPad uses:
+
+.. code:: bash
+
+   git worktree add www/common/onlyoffice/dist/v1 4f370beb
+   git worktree add www/common/onlyoffice/dist/v2b d9da72fd
+   git worktree add www/common/onlyoffice/dist/v4 6ebc6938
+   git worktree add www/common/onlyoffice/dist/v5 88a356f0
+   git worktree add www/common/onlyoffice/dist/v6 abd8a309
+   git worktree add www/common/onlyoffice/dist/v7 9d8b914a
+
+To install x2t, needed for document conversions, you should download `x2t.zip` from https://github.com/cryptpad/onlyoffice-x2t-wasm/releases, and extract its contents into `www/common/onlyoffice/dist/v7/x2t/`.
```

Comment:
Isn't it supposed to be `www/common/onlyoffice/dist/x2t/` (without `v7`in the path)?

Reply:
Ah, thanks for seeing this!